include/deal.II/lac/read_write_vector.h

```diff
@@ -206,6 +206,21 @@ namespace LinearAlgebra
     virtual void reinit (const IndexSet &locally_stored_indices,
                          const bool      omit_zeroing_entries = false);
 
+
+#ifdef DEAL_II_WITH_TRILINOS
+#ifdef DEAL_II_WITH_MPI
+    /**
+     * Initialize this ReadWriteVector by supplying access to all locally available
+     * entries in the given ghosted or non-ghosted vector.
```

Comment:
that makes it sound like you get a view, but don't we copy the elements?

Reply:
That is a good point. It is currently implemented as a copy but we might want to change that in the future. If ReadWriteVector supports not owning the data in the future, we could provide a view instead. I will change the comment.

Reply:
Yes, and we can then update the comment again.